alerts/src/iosMain/kotlin/Alerts.kt

```diff
@@ -32,14 +35,31 @@ actual class AlertInterface(
     private val parent: UIViewController
 ): BaseAlertPresenter(alert) {
 
-    override fun show(animated: Boolean, completion: (() -> Unit)?) {
+    override fun show(animated: Boolean, completion: (() -> Unit)) {
```

Comment:
This is identical code to https://github.com/splendo/kaluga/pull/37/files#diff-5709754ca8eba3cacbf6b495e1485457R40 can we not share it somehow?

Reply:
Good point